.eslintrc.js

```diff
@@ -4,7 +4,9 @@ module.exports = {
   "rules": {
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/prefer-stateless-function": [0],
-    "react/prop-types": [2, { ignore: [ 'history' ]}]
+    "react/prop-types": [2, { ignore: [ 'history' ]}],
+    "no-underscore-dangle": ["error", { "allow": ["__REDUX_DEVTOOLS_EXTENSION_COMPOSE__"] }],
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
```

Comment:
It's not the final solution. https://trello.com/c/RFLBpjv1/17-eslint

Reply:
I think, that now is ok https://github.com/hrodberaht/react-learning/commit/3f747bd683566c18e72c24ab034db684287a6f21

Reply:
It awesome 👍